controller/scheduler/job.go

```diff
@@ -87,6 +87,10 @@ type Job struct {
 	// referenced from within the main scheduler loop
 	State JobState `json:"state"`
 
+	// statusDescription is a human readable description of why a job
+	// is in a certain state
+	StatusDescription string
```

Comment:
Should this be an enum? It looks like all the current descriptions are basically status codes.